src/main/java/com/snowflake/kafka/connector/internal/SnowflakeIngestionServiceV1.java

```diff
@@ -254,4 +257,107 @@ public void close() {
   public SimpleIngestManager getIngestManager() {
     return this.ingestManager;
   }
+
+  /**
+   * configure the Snowpipe client and return the client sequencer
+   *
+   * @return a Long value contains the client sequencer
+   */
+  public Long configureClient() {
```

Comment:
should we return `ConfigureClientResponse` directly? let the caller decide what to use!

Reply:
Are we just using SnowflakeIngestService as a wrapper? All other apis have some extra logic to make them easier to use. For example, check history convert the response to map. I think it makes sense that we unwrap the response here in the ingestionService. The caller of this function does not need to know what structure of response we received from the SDK. Whenever we update the SDK, we only need to update those functions. This will make maintenance easier.

Reply:
Makes sense based on what is done earlier. Here is the case where I thought returning the response as is would be beneficial. Lets say we combine configure and get api to just configure api and then the configure api also returns offsetToken. In this case having returned the response object would also include offsetToken. And the response object type (`ConfigureClientResponse`) wont change, hopefully only more fields will be added.

Reply:
This also makes sense. Sure, we can return the entire response to caller.